src/parser/parseSVGDocument.ts

```diff
@@ -41,26 +41,13 @@ export function parseSVGDocument(doc, callback, reviver, parsingOptions) {
   }
   parseUseDirectives(doc);
 
-  let svgUid = uid(),
-    i,
-    len,
+  const svgUid = uid(),
     options = applyViewboxTransform(doc),
     descendants = Array.from(doc.getElementsByTagName('*'));
   options.crossOrigin = parsingOptions && parsingOptions.crossOrigin;
   options.svgUid = svgUid;
   options.signal = parsingOptions && parsingOptions.signal;
 
-  if (descendants.length === 0 && isLikelyNode) {
-    // we're likely in node, where "o3-xml" library fails to gEBTN("*")
-    // https://github.com/ajaxorg/node-o3-xml/issues/21
-    descendants = doc.selectNodes('//*[name(.)!="svg"]');
-    const arr = [];
-    for (i = 0, len = descendants.length; i < len; i++) {
-      arr[i] = descendants[i];
-    }
-    descendants = arr;
-  }
-
```

Comment:
didn't tests this yet

Reply:
I think that if SVG import test pass, it is ok.
The issue was not finding `getElementsByTagName('*')`